src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunReportPageTests.kt

```diff
@@ -55,12 +55,20 @@ class RunReportPageTests : SeleniumTest()
         assertThat(matches.size).isEqualTo(1)
     }
 
-    @Test
-    fun `can view git commits`()
+    @ParameterizedTest
+    @EnumSource(ConfigType::class)
+    fun `can view git commits`(configType: ConfigType)
```

Comment:
Maybe worth updating the test name now
```suggestion
    fun `can view git commits only when git allowed`(configType: ConfigType)
```